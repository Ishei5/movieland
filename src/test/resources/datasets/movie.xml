<dataset>
    <movieland_test.movie id="1" name_russian="Побег из Шоушенка" name_native="The Shawshank Redemption"
                          description="test" year_of_release="1994-01-01" rating="8.89" price="123.45"
                          picture_path="https://images-na.ssl-images-amazon.com/images/M/MV5BODU4MjU4NjIwNl5BMl5BanBnXkFtZTgwMDU2MjEyMDE@._V1._SY209_CR0,0,140,209_.jpg"/>
    <movieland_test.movie id="2" name_russian="Зеленая миля" name_native="The Green Mile" year_of_release="1999-01-01"
                          description="test" rating="8.88" price="134.67"
                          picture_path="https://images-na.ssl-images-amazon.com/images/M/MV5BMTUxMzQyNjA5MF5BMl5BanBnXkFtZTYwOTU2NTY3._V1._SY209_CR0,0,140,209_.jpg"/>
</dataset>
