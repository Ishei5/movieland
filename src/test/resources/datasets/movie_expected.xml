<dataset>
    <movieland_test.movie id="1" name_russian="Побег из Шоушенка" name_native="The Shawshank Redemption"
                          description="test" year_of_release="1994-01-01" rating="8.89" price="123.45"
                          picture_path="https://images-na.ssl-images-amazon.com/images/M/MV5BODU4MjU4NjIwNl5BMl5BanBnXkFtZTgwMDU2MjEyMDE@._V1._SY209_CR0,0,140,209_.jpg"/>
    <movieland_test.movie id="2" name_russian="Зеленая миля" name_native="The Green Mile" year_of_release="1999-01-01"
                          description="test" rating="8.88" price="134.67"
                          picture_path="https://images-na.ssl-images-amazon.com/images/M/MV5BMTUxMzQyNjA5MF5BMl5BanBnXkFtZTYwOTU2NTY3._V1._SY209_CR0,0,140,209_.jpg"/>
    <movieland_test.movie id="3" name_russian="Форрест Гамп" name_native="Forrest Gump"
                          description="test" year_of_release="1994-01-01" rating="8.6" price="200.60"
                          picture_path="https://images-na.ssl-images-amazon.com/images/M/MV5BNWIwODRlZTUtY2U3ZS00Yzg1LWJhNzYtMmZiYmEyNmU1NjMzXkEyXkFqcGdeQXVyMTQxNzMzNDI@._V1._SY209_CR2,0,140,209_.jpg"/>
    <movieland_test.movie id="4" name_russian="Список Шиндлера" name_native="Schindler''s List" year_of_release="1993-01-01"
                          description="test" rating="8.7" price="150.50"
                          picture_path="https://images-na.ssl-images-amazon.com/images/M/MV5BNDE4OTMxMTctNmRhYy00NWE2LTg3YzItYTk3M2UwOTU5Njg4XkEyXkFqcGdeQXVyNjU0OTQ0OTY@._V1._SX140_CR0,0,140,209_.jpg"/>
</dataset>
